evaluations/yieldscan_1_mmagician.md

```diff
@@ -0,0 +1,44 @@
+|    0d. | Article/Tutorial           | <ul><li>[x] </li></ul> | <ul><li>[YieldScan Help Center](https://intercom.help/yieldscan)</li></ul>                                                                                                                                                                                                                                                                                                           | Tutorial videos are helpful                                                                                                                                   |
+|     1. | Controller Account Support | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     2. | Ledger Wallet Support      | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     3. | Docker                     | <ul><li>[x] </li></ul> | <ul><li>[Docker file](https://hub.docker.com/r/sahilnanda/yieldscan-frontend)</li><li>[Docker command](https://github.com/yieldscan/yieldscan-frontend/tree/develop#docker)</li></ul>                                                                                                                                                                                                |                                                                                                                                                               |
+
+## General notes
+
+A lot of code duplication between the crawler and the api repos, probably everything under `src/models` & `src/interfaces` & large parts of other sub-directories. Would be good to consider using for example git submodules to share the code.
+
+Could you also point to the exact location of the patch used for going around ledger's batch Tx's please?
```

Comment:
if you mean the patch we used for handling ledger's unsupported staking batch transactions, here's the link to the module: https://github.com/yieldscan/yieldscan-frontend/tree/master/components/staking/stepperSignerPopover

Let me know if this is not the information you were looking for.

Reply:
That's it, thanks!